packages/config/config/devices/0x013c/templates/philio_template.json

```diff
@@ -36,10 +36,12 @@
 			}
 		]
 	},
-	"auto_report_interval": {
+	"battery_report_interval_30m": {
```

Comment:
The point with the new templates was that it's not only battery that is reported automatically with this pattern. I was planning to use this for all auto reports. I don't think this change was good.

Reply:
I see. Will rename it then.